scripts/stripnulls.sh

```diff
@@ -7,7 +7,7 @@
 SED=$(command -v gsed)
 SED=${SED:-"sed"}
 
-FILES=$(grep -rl '"schemaVersion": 3[0123]' devenv)
+FILES=$(grep -rl '"schemaVersion": 3[01234]' devenv)
```

Comment:
```suggestion
FILES=$(grep -rl '"schemaVersion": 3[34]' devenv)
```

Reply:
With the changes to the schema, we're not actually supporting those older versions anymore, so might as well narrow the margin for error with the hacky shell script.